src/plugins/testbed/server/index.ts

```diff
@@ -41,6 +41,11 @@ export const config: PluginConfigDescriptor = {
     uiProp: true,
   },
   schema: configSchema,
+  deprecations: ({ rename, unused, renameFromRoot }) => [
+    rename('securityKey', 'secret'),
+    renameFromRoot('oldtestbed.uiProp', 'testbed.uiProp'),
+    unused('deprecatedProperty'),
+  ],
```

Comment:
Added an example in testbed. Not sure if I should keep it.

Reply:
we could add something similar to the integration tests in LP https://github.com/restrry/kibana/blob/55604f711f2b06ca58235c1aabc45f5281e13642/src/legacy/server/config/__tests__/deprecation_warnings.js#L28
although it tests only logging part

Reply:
Do we want a similar approach to what LP does, by actually spawning a real kibana process? Could find a NP equivalent to legacy `const RUN_KBN_SERVER_STARTUP = require.resolve('./fixtures/run_kbn_server_startup');` ? Seems like more than what we call `integration_test` in NP?

Reply:
`run_kbn_server_startup` uses `test_utils` under the hood. So do we in the integrations tests. 
https://github.com/restrry/kibana/blob/55604f711f2b06ca58235c1aabc45f5281e13642/src/core/server/http/integration_tests/core_services.test.ts#L23
so yes, I think we can adopt the same approach

Reply:
Thanks, missed that.